lib/rules/one-component-per-file.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @fileoverview enforce that each component should be in its own file
+ * @author Armano
+ */
+'use strict'
+const utils = require('../utils')
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'enforce that each component should be in its own file',
+      category: undefined, // strongly-recommended
+      url: 'https://github.com/vuejs/eslint-plugin-vue/blob/v5.0.0-beta.4/docs/rules/one-component-per-file.md'
+    },
+    fixable: null,
+    schema: [],
+    messages: {
+      toManyComponents: 'There is to many components in this file.'
```

Comment:
I think we should write : There is more than one component in this file.

Reply:
Also this should be `tooManyComponents`.